src/App.tsx

```diff
@@ -1,10 +1,42 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Footer, TodoStatus } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { Errors } from './components/Errors/Errors';
+
+const getVisibleTodos = (todos: Todo[], status: TodoStatus) => {
```

Comment:
This function should be moved to a separate file

Reply:
> This function should be moved to a separate file

Good point